stable_baselines/common/callbacks.py

```diff
@@ -50,6 +50,16 @@ def init_callback(self, model: 'BaseRLModel') -> None:
         self.logger = logger.Logger.CURRENT
         self._init_callback()
 
+    def update_locals(self, locals_:Dict[str, Any]):
+        '''
```

Comment:
Please use double quotes and do not forget `:param locals_` in the docstring
The return type is also missing...